content/blog/2010/2010-02-25-spotlight-on-springsource.adoc

```diff
@@ -9,7 +9,7 @@
   - interview
 :author: rtyler
 ---
-For this week's user spotlight segment, I'm talking with https://twitter.com/dougmaceachern[Doug MacEachern] of https://www.crunchbase.com/company/hyperic[Hyperic], part of https://www.crunchbase.com/company/springsource[SpringSource], a division of VMware, hoping I got that dependency chain correct. Hyperic builds enterprise systems monitoring and management software and also contributes to a number of open source projects, many of which are built with Hudson. image:http://blog.hudson-ci.org/sites/default/files/hyperic_hudson.png[A small subsection of Hyperic's Hudson,350,link=http://blog.hudson-ci.org/sites/default/files/hyperic_hudson.png]
+For this week's user spotlight segment, I'm talking with https://twitter.com/dougmaceachern[Doug MacEachern] of https://www.crunchbase.com/company/hyperic[Hyperic], part of https://www.crunchbase.com/company/springsource[SpringSource], a division of VMware, hoping I got that dependency chain correct. Hyperic builds enterprise systems monitoring and management software and also contributes to a number of open source projects, many of which are built with Hudson. image:https://blog.hudson-ci.org/sites/default/files/hyperic_hudson.png[A small subsection of Hyperic's Hudson,350,link=http://blog.hudson-ci.org/sites/default/files/hyperic_hudson.png]
```

Comment:
domain doesn't exist
```suggestion
For this week's user spotlight segment, I'm talking with https://twitter.com/dougmaceachern[Doug MacEachern] of https://www.crunchbase.com/company/hyperic[Hyperic], part of https://www.crunchbase.com/company/springsource[SpringSource], a division of VMware, hoping I got that dependency chain correct. Hyperic builds enterprise systems monitoring and management software and also contributes to a number of open source projects, many of which are built with Hudson.
```

Reply:
Yea your probably right that's the better fix, but I'm going to leave it open for now. I don't know how important it is to preserve old data.

Reply:
I don't think we should keep dead links around 😄 .

Reply:
@MarkEWaite @MarkEWaite @MarkEWaite

Reply:
I agreed that we should not keep dead links.